fxa-auth-db-server/docs/DB_API.md

```diff
@@ -249,7 +250,7 @@ Parameters.
 
 Each token takes the following fields for it's create method respectively:
 
-* sessionToken : data, uid, createdAt
+* sessionToken : data, uid, createdAt, userAgent, os, deviceType
```

Comment:
> I'm not currently recording any version information because instinctively it didn't feel that useful, 
> especially for browsers with rapid release cycles.

I think it'll be useful.  As a point of comparison, out kibana dashboards [1] keep coarse version information like "Windows 7" and "Firefox 39" which seems like a nice balance.

[1] e.g. https://kibana.fxa.us-west-2.prod.mozaws.net/index.html#/dashboard/file/auth_http_status.json

> deviceType is not currently the same as what we discussed briefly last week.

From a brief look, it seems like this is almost entirely "mobile device type" and that desktop machines just get "Other".  If so, maybe we can record it as such and just say "anything with a deviceType field is a mobile device".  I don't know how well that maps onto the model provided by the lib though.

> I briefly wondered about normalising this information in a separate table [...] so what's the point, really?

I slept on it, and I came to a similar conclusion, there's not really much point as it's only a handful of fields that I think are unlikely to grow.